extensions/script/python/ExecutePythonProcessor.h

```diff
@@ -34,6 +34,8 @@
 #include "PythonScriptEngine.h"
 #include "core/Property.h"
 
+#pragma GCC visibility push(hidden)
```

Comment:
Does this solve the `declared with greater visibility than the type of its field` error? If so, how does this work?
I found these refs: [[1]](https://github.com/vgc/vgc/issues/11), [[2]](https://stackoverflow.com/questions/49252686/exported-class-with-non-exported-data-member?newreg=b2adafbe461c42faa7888ff23b6b05c3) on the issue; how does this relate?

Reply:
Yes that's the error and those refs are related. There was a similar fix by @szaszm previously in #754, he also mentioned that the fix is due to pybind11 requiring `-fvisibility=hidden`. This solution is now applied in a wider scope by including the classes using PyProcessSession.